src/libraries/Microsoft.Extensions.Configuration.FileExtensions/src/FileLoadExceptionContext.cs

```diff
@@ -10,6 +10,12 @@ namespace Microsoft.Extensions.Configuration
     /// </summary>
     public class FileLoadExceptionContext
     {
+        public FileLoadExceptionContext(FileConfigurationProvider provider, Exception exception)
```

Comment:
This is a breaking change (as noted in the build break). The class used to have a parameterless ctor, and now it no longer does. Also we can't be adding new public API without going through API review. Please revert this.